UI/src/components/widgets/deploy/module.js

```diff
@@ -21,6 +21,29 @@
 
     angular
         .module(HygieiaConfig.module)
+        .directive('validregex', function() {
+          return {
+            require: 'ngModel',
+            link: function(scope, elm, attrs, ctrl) {
+              ctrl.$validators.validregex = function(modelValue, viewValue) {
+                if (ctrl.$isEmpty(modelValue)) {
```

Comment:
Do we need this check? I think the RegExp(..) will handle the empty scenario too?